.storybook/util/theme.js

```diff
@@ -86,7 +86,7 @@ export const components = {
     size: TEXT_SIZE,
     css: italicStyles,
   }),
-  ul: withThemeProvider(List, { size: TEXT_SIZE }),
-  ol: withThemeProvider(List, { size: TEXT_SIZE, variant: 'ordered' }),
+  ul: withThemeProvider(List),
+  ol: withThemeProvider(List, { variant: 'ordered' }),
```

Comment:
Which issue does this address? 🤔

Reply:
If you check the [deployed Storybook](https://oss-circuit-ui-git-next-sumup.vercel.app/?path=/story/introduction-contributing-overview--page) for `next` for example, there's an error in pages where markdown lists were used, because the `List` component doesn't accept a `TEXT_SIZE` anymore (the text sizes changed to `one`/`two` and the List still expects a `kilo`/`mega`). I took it out because `mega` is the default

Reply:
Oh, that's an oversight in the List component then! It should accept the same sizes as the Body component.

Reply:
@amelako, would you mind updating the sizes for the List component and adding it to the typography sizes codemod?